<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(124, 174, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Perpetua</family>
      <pointsize>35</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 35pt &quot;Perpetua&quot;;</string>
    </property>
    <property name="text">
     <string>Pharmacy Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 212, 255);</string>
    </property>
    <property name="text">
     <string>Back to Login</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_pharmacy_name">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pharmacy_website">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Website:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pharmacy_address">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pharmacy_owner">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Owner:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pharmacy_phone">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Phone Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PHours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pharmacy Working Hours:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_monday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tuesday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wednesday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_thursday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_friday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Friday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_saturday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturday: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sunday_hours">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunday: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="hoursUpdateBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 212, 255);</string>
    </property>
    <property name="text">
     <string>Edit Store Information</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/pharm.qrc"/>
 </resources>
 <connections/>
</ui>
